app/templates/views/subscription.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+{% block title %}Subscription{% endblock %}
+
+{% block content %}
+
+<div class="container-fluid padding">
+  <div class="row padding text-center ">
+    <div class="col-xs-12 col-sm-12 col-md-6">
+       <h2> Click submit to subscribe to the newsletter:</h2><br>
```

Comment:
I've had a word with the team at New Acroopolis, and they have asked for a few changes to the wording:

Change `newsletter` be changed to `magazine` and `Submit` to `Subscribe`

And also can the whole phrase be changed to - 

`By clicking subscribe you will receive our magazine and information about our events:`